tests/conftest.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+import pytest
+
+def pytest_addoption(parser):
+    parser.addoption("--local", action="store_true", default=False, help="run local tests")
+    parser.addoption("--dataset", type=str, default="", help="dataset to test locally")
+    parser.addoption("--skip-download", action="store_true", default=False, help="skip download step")
+
+
+# @pytest.fixture(scope='session')
+# def local(request):
+#     return request.config.getoption('--local')
+
+@pytest.fixture(scope='session')
+def skip_local(request):
+    if request.config.getoption('--local'):
```

Comment:
Maybe this is right but is just a bit unintuitive for me: you skip the local tests if you get the `--local` flag and skip the remote if you don't? or I completely misunderstood?

Reply:
It's confusingly written. The `--local` just means that it's being run locally. Then there are two fixtures `skip_local` and `skip_remote` which can be passed to different tests. When a test is passed `skip_local`, it will only skip it when `--local` is passed. When a test is passed `skip_remote`, it will only skip it when `--local` is not passed.